### PR TITLE
Update URLs for the management and backend parts.

### PR DESCRIPTION
**Pull request by duncanparkes**
Merge all the backend urls into a 'management' section, and fix all the errors which were revealed/created.

Contrary to the name of the branch, this deals primarily with urls for a logged in user to manage their instance. Further url moving will happen later, but should hopefully be much less painful.

### CODE REVIEW

**Comment by lfalvarez** [comment]
wooohoooo!!! :+1:

**Comment by duncanparkes** [comment]
This has now been rebased over master and all the tests are passing - woop!

**Comment by duncanparkes** [comment]
@lfalvarez It would be great if you could sanity check this one.

File: nuntium/urls.py [hunk 1]
```diff
@@ -10,6 +34,27 @@
...
 
+    url(r'^confirm_message/(?P<slug>[-\w]+)/?$', ConfirmView.as_view(), name='confirm'),
+
+    url(r'^moderation_accept/(?P<slug>[-\w]+)/?$', AcceptModerationView.as_view(), name='moderation_accept'),
+    url(r'^moderation_reject/(?P<slug>[-\w]+)/?$', RejectModerationView.as_view(), name='moderation_rejected'),
+
+)
 
-    url(r'^accounts/login/$', login, name='account_login')
+managepatterns = patterns('',
```
Comment (lfalvarez) [inline nuntium/urls.py]: I just loved this, but I'm wondering if it makes sense to have this patterns over at user_section/urls.py? At some point I thought of the user_section module as the management module (of course everything is changeable).

**Pull request merged**